views/layouts/settings.hbs

```diff
@@ -1,18 +1,13 @@
-<html lang="en">
+<!DOCTYPE html>
+<html lang="{{lang}}">
   <head>
     {{> standard_headers}}
     <link type="text/css" rel="stylesheet" href="content-layout.css" />
   </head>
   <body>
     <main data-testid="app-express">
       <div class="app-container">
-        {{{nav}}}
-        <div class="container spacing-32">
-          <div class="content">
-            {{{body}}}
-          </div>
-          {{> ory_branding}}
```

Comment:
is the Ory branding now included in the `UserSettingsScreen` component?

Reply:
Yes, because I needed to move the wrappers there as well. Unless I can render the nav in the layout already :thinking: